api/python/notebooks/experimental/pytorch.ipynb

```diff
@@ -810,9 +632,7 @@
    "cell_type": "markdown",
    "id": "c2fd35fa",
    "metadata": {},
-   "source": [
-    "As we can see, the predictions are poor. It is as left as an exercise to the reader to find a better performing model."
-   ]
+   "source": []
   }
  ],
  "metadata": {
```

Comment:
there seems to be an empty cell at the end of the notebook

Reply:
🔥